app/views/hyrax/file_sets/_show_details.html.erb

```diff
@@ -0,0 +1,23 @@
+<h2>File Details</h2>
+<dl class="dl-horizontal file-show-term file-show-details">
+  <dt>Depositor</dt>
+  <dd itemprop="accountablePerson" itemscope itemtype="http://schema.org/Person"><span itemprop="name"><%= link_to_profile @presenter.depositor %></span></dd>
+  <dt>Date Uploaded</dt>
+  <dd itemprop="datePublished"><%= @presenter.date_uploaded %></dd>
+  <dt>Date Modified</dt>
+  <dd itemprop="dateModified"><%= @presenter.date_modified %></dd>
+  <% if @presenter.extracted_text? %>
+  <dt>OCR</dt>
+  <dd><%= link_to 'Download', "/downloads/#{@presenter.id}?file=extracted_text" %></dd>
```

Comment:
is there a more orthodox way to build this URL?  That's a genuine question; I don't object to what's here currently

Reply:
Not sure of a better way to form URL, but I moved the link creation to the presenter so all extracted_text stuff resides in same file.